### ISSUE TITLE
Keyboard focus is lost on tab navigating into ComboBox with no selection

### ISSUE THREAD

**Issue by arjunbalgovind** [microsoft/microsoft-ui-xaml#3503]
**Describe the bug**

If you tab to a ComboBox which doesn't have a selection and you press Enter, the keyboard focus is lost and requires you to press down arrow key in order to get focus on the first element.

This causes another issue if you have many items in the ComboBox (more than the MaxDropDownHeight), then when you open the ComboBox the middle of the ComboBox item list is visible rather than the first item.

**Steps to reproduce the bug**

The issue can be reproduced on Xaml Controls Gallery by commenting out this line https://github.com/microsoft/Xaml-Controls-Gallery/blob/22c5fd1b7d72b02b9a7885232c22bcb3e92d3251/XamlControlsGallery/ControlPages/ComboBoxPage.xaml.cs#L77
1. Tab to the ComboBox without selection
2. Press Enter
**Expected behavior**

Keyboard focus should be on the first element as there is no selection

**Screenshots**

Focus issue:
![winUIDropDownFocus](https://user-images.githubusercontent.com/32061677/97356899-2f045800-1856-11eb-90cd-9c626f1cad0f.gif)
Middle of the list visible by default and not the top:
![winUIDropDownMiddle](https://user-images.githubusercontent.com/32061677/97356903-2f9cee80-1856-11eb-864c-3c231ee3e837.gif)

**Version Info**

NuGet package version: 

Microsoft.UI.Xaml 2.5.0-prerelease.200923002
Can also be reproduced with winRT Xaml Islands

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
|  |  |

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2020 Update (19041)            | Yes   |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**

The issue can be reproduced on PowerToys Keyboard Manager https://github.com/microsoft/PowerToys/issues/7070

**Comment by shaheedmalik** [comment]
Does this happen on the Touch Keyboard?

**Comment by arjunbalgovind** [comment]
I have not tried it with the Touch keyboard, but it happens on normal physical keyboards as well as the On-Screen keyboard. I added the use of On-Screen keyboard in the gif so that the repro steps are clear.

**Comment by llongley** [comment]
Will be fixed in the next version of WinUI 3.

**Comment by crutkas** [comment]
@llongley is that 1.1?

**Comment by MikeHillberg** [comment]
Yes, it will be in 1.1-preview3
